Bottom: d6b2f9ef3354d53203c96a3677cd3256d18c3c9e
Top:    a246069b7a7a8ac885186f89c20b45ed61a54ffa
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 17:08:18 -0500

Refresh of add-environment-variables-for

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index c0988cfb..df2f075c 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -38,6 +38,9 @@ container_env:
   BUILDBOT_DATABASE: 'sqlite:///{{ data_path }}/state.sqlite'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
   BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
+  BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
+  BUILDBOT_SMTP_USER: 'Dean.cox'
+  BUILDBOT_SMTP_PASS: 'no5156'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
